plans/software_today.markdown

```diff
@@ -0,0 +1,106 @@
+
+
+# Critical Challenges Planning
+
+## The Challenge
+
+* What Critical Challenge are you going to work on?
+
+We need the Today app to make it easy for students and staff to know where they should be and when.
+
+* Why is this a “challenge”?
+
+Schedules are hard, and things are very fluid.
+
+* Can it be solved? What would be value to the organization/team in solving it or working towards a solution?
+
+It can be improved, I don't think it can be solved. Calendaring is hard, and proof of that is the lack of really good
+commercial calendaring software.
+
+* What happens if we fail to solve it / improve it?
+
+We have the status quo, with occasional bits of confusion.
+ 
+* What skills do you have that make you the right person to work on this challenge?
+
+I've been doing the calendar this module.
+
+## The Past
+
+* What’s been done so far?
+
+There are a number of shared calendars, and I've been trying to build out the entirety of the next week, so people have at least 
+a rough idea of plans for the upcoming week.
+ 
+* Was it effective? How do you know?
+
+It works well enough. People figure out where they are supposed to be.
+
+* Was this someone’s challenge before you? Why did they stop doing it or how did we make progress without addressing it?
+
+Steve.
+
+## The Present
+
+* What are the first steps that you’re ready to execute now?
+
+I would like to have a deadline of say Thursday or so for the schedule for the upcoming week to have a good amount of time to prep
+Today for the upcoming week. This is, as always, a soft deadline as to give staff the ability to call 
+[audibles](https://en.wikipedia.org/wiki/Quarterback#Special_tactics). 
+
+* What will be the next three things you do?
+
+Look for ways to improve the Today app. Ask for schedules earlier. Continue to do schedules a week in advance.
+
+* How will you keep track of progress and results?
+
+A focus group would probably be pretty helpful.
+
+* Are you empowered to take the actions you think necessary?
+
+Pull requests for days. Yes.
+
+
+## The Future
+
+* What would total success look like?
+
+No more double bookings, less confusion.
```

Comment:
I think the way you've been doing it so far, where one module is generally always in the same room, works pretty well.